pluto_build/sql/qaqc_outlier.sql

```diff
@@ -0,0 +1,38 @@
+-- -v VERSION=$VERSION
+DROP TABLE IF EXISTS qaqc_outlier;
```

Comment:
This should come out as going forward the QAQC should append new records

Reply:
Missed this update in my comment yesterday, but this also needs to come out before merging

Reply:
Yea, thanks for pointing this out. I forgot to take it out before and just deleted it in the latest commit.